.github/workflows/os-smoke-test.yml

```diff
@@ -0,0 +1,34 @@
+name: OS Support Smoke Test
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  build:
+    name: Build Zeebe and run smoke test on ${{ matrix.os }} to ensure development support.
```

Comment:
This name is pretty long, especially when viewed in the Pull Request Checks in the bottom of a pull request.

🔧 Please find a shorter name